### DISCUSSION TITLE
Use "." as default-value for the --workspace-folder argument

### DISCUSSION THREAD

**Discussion by exhuma** [devcontainers/spec#308] [Ideas]
Considering `@devcontainers/cli`: Currently many commands require me to explicitly set `--workspace-folder`. This is redundant and cumbersome. I would wager that the "current folder" is a sensible default for most. Thus it would greatly improve usability if the argument would be *optional* with the default set to `.`

**Comment by samruddhikhandale** [comment]
Hi 👋 

We have a similar request added to https://github.com/devcontainers/cli/issues/29. Do you mind bumping it up with another comment? Thanks!
